src/utils/performance.ts

```
@@ -0,0 +1,115 @@
+    console.groupEnd();
+  }
+}
+
+// Export a singleton instance
+export const performanceMonitor = new PerformanceMonitor();
+
+/**
+ * Decorator function to automatically time async functions
+ */
+export function timed(operation: string) {
+  return function <T extends (...args: any[]) => Promise<any>>(
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)